### ISSUE TITLE
How to train with TalkingHead-1KH?

### ISSUE THREAD

**Issue by yahskapar** [zhanglonghao1992/One-Shot_Free-View_Neural_Talking_Head_Synthesis#65]
Has anyone here managed to train using this repo and the TalkingHead-1KH dataset? I believe the author (@zhanglonghao1992) has trained on this dataset before, but the code in the repo seems to default to training with VoxCeleb. It's not clear to me how to use the TalkingHead-1KH dataset out-of-the-box with this code. Should additional pre-processing steps be done, similar to those done to VoxCeleb [here](https://github.com/AliaksandrSiarohin/video-preprocessing)? Would you be willing to elaborate on what you did exactly to get, I assume, reasonable results when compared to VoxCeleb, @zhanglonghao1992?

**Comment by zhanglonghao1992** [comment]
@yahskapar Hi~
Just process the data according to https://github.com/AliaksandrSiarohin/video-preprocessing/blob/master/crop_vox.py. You only need to set the division of training set and test set.

**Comment by yahskapar** [comment]
Thanks for the reply, I will try to process according to that crop_vox.py script.
